apps/ui/components/calendar/ReservationCalendarControls.tsx

```diff
@@ -651,25 +652,25 @@ const ReservationCalendarControls = <T extends Record<string, unknown>>({
     </SubmitButtonWrapper>
   ) : null;
 
+  const TogglerLabelContent = areControlsVisible ? (
+    <div>&nbsp;</div>
+  ) : (
+    <>
+      <TogglerDate>{togglerLabel}</TogglerDate>
+      <TogglerPrice>
+        {t("reservationUnit:price")}: {price}
+      </TogglerPrice>
+    </>
+  );
```

Comment:
Generally bad. (yeah the above one is also bad).
Guess the linter rules don't block defining components inside components but they should.

Moving it is good, just not sure we want the downside.

Every time a component is rendered it creates a new component, destroys the old one, swaps the new into the virtual dom, which recreates the dom matching that component.

Reply:
Good point. Moved it outside the ReservationCalendarControls component.